engine/classes/Elgg/Database/RelationshipsTable.php

```diff
@@ -117,11 +117,8 @@ function add($guid_one, $relationship, $guid_two) {
 		if ($id !== false) {
 			$obj = get_relationship($id);
 	
-			// this event has been deprecated in 1.9. Use 'create', 'relationship'
-			$result_old = _elgg_services()->events->trigger('create', $relationship, $obj);
-	
 			$result = _elgg_services()->events->trigger('create', 'relationship', $obj);
```

Comment:
I agree event/hook naming is rather arbitrary. Another reason for coming up with a solid convention before adding more. Creating it today I'd use a single string and allow simple pattern matching.